fog-updater/src/test_util.py

```diff
@@ -0,0 +1,82 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+
+from fog_update import eval_extract, swap_file_list
+
+
+REPOSITORIES_YAML = """
+---
+version: "2"
+applications:
+  - app_name: firefox_desktop
+    metrics_files: # When adding here, consider if you should also add to pine.
```

Comment:
Are you keeping this comment because it's hilarious?

Reply:
depends on who you ask! (Nah, I copied that from the original a while back in the before times)